WcaOnRails/app/views/admin/compute_auxiliary_data.html.erb

```diff
@@ -11,20 +11,24 @@
         <li class="list-group-item">Clears some cache files (living on the PHP side)</li>
       </ul>
     </div>
-    <% if ComputeAuxiliaryData.in_progress? %>
-      <%= alert :info, "The data is being computed. Thanks for checking =)" %>
-    <% elsif ComputeAuxiliaryData.finished? %>
-      <%= alert :success do %>
-        Data was last computed <%= time_ago_in_words ComputeAuxiliaryData.end_date %> ago
-        and took <%= distance_of_time_in_words ComputeAuxiliaryData.start_date, ComputeAuxiliaryData.end_date %>
-      <% end %>
+    <% if Result.exists?(personId: "") %>
+      <%= alert :warning, "Some results may be missing their corresponding WCA ID, which means that someone hasn't finished submitting results." %>
```

Comment:
No need to sound unsure here, be confident and say "Some results **are** missing their corresponding WCA ID"!